client/extensions/woocommerce/state/sites/setup-choices/reducer.js

```diff
@@ -6,6 +6,7 @@ import { LOADING } from 'woocommerce/state/constants';
 import {
 	WOOCOMMERCE_SETUP_CHOICES_REQUEST,
 	WOOCOMMERCE_SETUP_CHOICES_REQUEST_SUCCESS,
+	WOOCOMMERCE_SETUP_CHOICE_UPDATE_REQUEST_SUCCESS,
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/15223/commits/004863a3c5f48ce79a2d0ea93ba459112536c873